.github/workflows/pytest.yml

```
@@ -0,0 +1,34 @@
+name: pytest
+
+# This workflow will
+# run all tests with pytest  
+on: [push]
+
+jobs:
+  
+  tests:
+   runs-on: ubuntu-18.04
+   strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        python-version: [3.6, 3.7, 3.8, 3.9]
+   steps:
+     
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}    
+
+    - name: Test python version [3.6, 3.7, 3.8, 3.9] and install dependencies
```

Not sure, but I think you can use a placeholder here:

```suggestion
    - name: Test python version ${{ matrix.python-version }} and install dependencies
```